libs/domain/checkout/billing-address/billing-address.component.ts

```diff
@@ -52,12 +59,13 @@ export class CheckoutBillingAddressComponent
         value: deliveryAddress,
       });
     }
-  });
+  };
 
   @query('oryx-checkout-address')
   protected checkoutAddress?: CheckoutAddressComponent;
 
-  protected override render(): TemplateResult {
+  protected override render(): TemplateResult | undefined {
```

Comment:
```suggestion
  protected override render(): TemplateResult | void {
```

I've been using void everywhere, but wasn't aware undefined could be used as well

Reply:
I'd use `void` when you don't return use return in the function (or result is not important). In this case it's debatable, as the name of the method suggests doing something (`render`) while technically it's just returning TemplateResult for the rendering logic. That's why I went with either TemplateResult or undefined.

Reply:
...but I actually got convinced, void looks bit nicier even in this context. And we don't write `return undefined` explicitely anyway.